src/boxx/CylinderBox.java

```diff
@@ -0,0 +1,40 @@
+package boxx;
+
+
+public class CylinderBox implements BoxInterface {
+    private double h;
+    private double r;
+    double price;
```

Comment:
How is about extracting price in order not to write the same field for all boxes separately?